code/modules/religion/burdened/psyker.dm

```diff
@@ -0,0 +1,305 @@
+/obj/item/organ/internal/brain/psyker
+	name = "psyker brain"
+	desc = "This brain is blue, split into two hemispheres, and has immense psychic powers. Why does that even exist?"
+	icon_state = "brain-psyker"
+	actions_types = list(
+		/datum/action/cooldown/spell/pointed/psychic_projection,
+		/datum/action/cooldown/spell/charged/projectile_booster,
+		/datum/action/cooldown/spell/forcewall/psychic_wall,
+	)
+
+/obj/item/organ/internal/brain/psyker/Insert(mob/living/carbon/inserted_into, special, drop_if_replaced, no_id_transfer)
+	if(!istype(inserted_into.get_bodypart(BODY_ZONE_HEAD), /obj/item/bodypart/head/psyker))
+		return FALSE
+	. = ..()
+	inserted_into.AddComponent(/datum/component/echolocation)
```

Comment:
I feel like there's definitely a way to use HARS to put your psyker brain in a body without a psyker head. maybe become a psyker -> receive HARS -> remove brain from body -> cure HARS in corpse -> reinsert brain into body as a chest organ -> behead yourself -> attach a new head?